testdata/flushable_ingest

```diff
@@ -618,3 +618,70 @@ L0.1:
   000004:[a#12,SET-b#12,SET]
 L0.0:
   000007:[a#10,SET-b#11,SET]
+
+close
+----
+
+reset
+----
+
+build small
+set-multiple 10 small
+----
+
+build large
+set-multiple 100000 large
+----
+
+batch
+set small-00001 before-ingest
+set large-00001 before-ingest
+----
+
+blockFlush
+----
+
+ingest small large
+----
+
+allowFlush
+----
+
+# When looking inside the small table, we will read the index block, then we
+# should read the bloom filter, followed by a data block read.
+get with-fs-logging
+small-00001
+----
+read-at(194, 27): 000004.sst
+read-at(120, 74): 000004.sst
+read-at(0, 120): 000004.sst
+small-00001:val-00001
+
+# When the key doesn't pass the bloom filter, we should see only two block
+# reads.
+get with-fs-logging
+small-00001-does-not-exist
+----
+read-at(194, 27): 000004.sst
+read-at(120, 74): 000004.sst
+small-00001-does-not-exist: pebble: not found
+
+# When looking inside the large table, we will not read the bloom filter which
+# is large. We read the top-level index block, a second-level index block, and a
```

Comment:
I'm catching this late, but the first read is probably a footer or meta-index read and not a two-level filter block read, right? Either way, doesn't block this PR.